cypher/cypher-docs/src/docs/dev/ql/administration/security/index.asciidoc

```diff
@@ -99,10 +106,11 @@ The special characters and their meaning are as follows:
 
 |===
 
-Any other characters do not have any special meaning, therefore not needing to be escaped.
+There are no other characters that need to be escaped.
```

Comment:
This still have @Lojjs 'problem' with using escaped in a completely different way then what we normally use in cypher (where things are escaped using `)

It's one of her comments on the 4.3 version

Reply:
ok, will look ahead at those review comments in #758